### PR TITLE
[CDI3] Clarify and improve introduction of Fourier Series

### PR DESCRIPTION
**Pull request by Calhau18**

### CODE REVIEW

**Comment by diogotcorreia** [comment]
@randomicecube I'll keep it, but it's definitely duplicate content, since the assumptions for `a_n` are valid for all `n`, including zero. It might help someone tho

File: content/cdi-iii/0010-serie-fourier.md [hunk 1]
```diff
@@ -49,29 +50,31 @@ $$
...
+
 :::
 
-A expressão resultante, $SF_f(x)$ vai corresponder exatamente a $f$ no intervalo $]-L, L[$, exceto onde a função tem pontos de descontinuidade.  
-Aliás, uma das grandes vantagens da Série de Fourier é que a função não precisa de ser contínua, apenas seccionalmente contínua (e ter derivada seccionalmente contínua).
+A expressão resultante, $SF_f(x)$ vai corresponder exatamente a $f$ nos pontos em $]-L, L[$ em que $f$ é contínua.
+Aliás, uma das grandes vantagens da Série de Fourier é que a função não precisar de ser contínua, apenas seccionalmente contínua (e ter derivada seccionalmente contínua).
 
-Nos pontos de descontinuidade e nas "pontas" do intervalo ($-L$ e $L$), a expressão da Série de Fourier equivale ao ponto médio entre os dois pontos adjacentes. Traduzindo isto para um sistema, obtemos o [**Teorema da Convergência Pontual da Série de Fourier**](color:green):
+Nos pontos de descontinuidade, a expressão da Série de Fourier equivale ao ponto médio entre os valores da função a vir de ambas as direções. Traduzindo isto para um sistema, obtemos o [**Teorema da Convergência Pontual da Série de Fourier**](color:green):
```
Comment (diogotcorreia) [inline content/cdi-iii/0010-serie-fourier.md:60]: @Calhau18 why did you remove `e nas "pontas" do intervalo ($-L$ e  $L$)`? Seems like you've removed key information

**Review by diogotcorreia — CHANGES_REQUESTED** [review]
I'm a bit confused by the changes to a_0, I think they're a downgrade, I'll discuss this with you and edit accordingly

File: content/cdi-iii/0010-serie-fourier.md (hunk 1 above)
Comment (Calhau18) [reply content/cdi-iii/0010-serie-fourier.md:60]: If I recall correctly, it's because the edges (L and -L) belong to the function domain and therefore are either continuity points or discontinuity points. 
Remember that Fourier Series are (because they are periodic) always defined in the whole real domain.
In either case, the theorem already specifies the series' value in such points, making this redundant information. 
If you consider however that it is worth keeping, even if as a parenthesis, please do. I think that probably is the right call.

**Review by diogotcorreia — APPROVED** [review]
